src/styles/extension.css

```diff
@@ -88,4 +88,21 @@
 
 .giphy-widget li:nth-child(odd) {
     margin-right: 2%;
+}
+
+/* Styles to override Refined GitHub and play nicely */
+.gifhub-button.toolbar-commenting {
+    display: block !important;
+}
+.gifhub-button .toolbar-group {
+    display: none;
```

Comment:
This style has the side-effect of hiding the markdown toolbar for non `Refined GitHub` users on commit page comments

Screen-shot is with `Refined` disabled
![image](https://cloud.githubusercontent.com/assets/5233399/14096555/5c406196-f52f-11e5-95f7-74362ceed27f.png)